dataactcore/models/lookups.py

```diff
@@ -0,0 +1,90 @@
+# This file defines a series of constants that represent the values used in the
+# broker's "helper" tables. Rather than define the values in the db setup scripts
+# and then make db calls to lookup the surrogate keys, we'll define everything
+# here, in a file that can be used by the db setup scripts *and* the application
+# code.
+# todo: replace getIdFromDict and getNameFromDict baseInterface functions with these constants
+
+from collections import namedtuple
+
+LookupType = namedtuple('LookupType', ['id', 'name', 'desc'])
+LookupFileType = namedtuple('LookupFileType', ['id', 'name', 'desc', 'letter', 'order'])
```

Comment:
Thanks for adding this file, it's nice to have all these values in one location.  Minor question on FILE_TYPE, I'm wondering about the purpose of the "order" element here, it doesn't seem to be used when loading the file_type table.  Related question, if we are adding an explicit ordering, should D1 be before D2 in that ordering?